src/cljs/ataru/hakija/application_handlers.cljs

```diff
@@ -263,14 +263,17 @@
 
 (defn set-question-group-row-amounts [db]
   (let [flattened-form-fields (cljc-util/flatten-form-fields (-> db :form :content))]
-    (reduce-kv (fn [db answer-key {:keys [value]}]
+    (reduce-kv (fn [db answer-key {:keys [value values]}]
```

Comment:
Extract large anon function?

Reply:
Discussed this face-to-face. Decided to leave this as-is for now.